infrastructure/files/game/Dockerfile.tpl

```diff
@@ -16,18 +16,37 @@
 # Build a container for the Unreal Engine dedicated game server
 #
 
-FROM ghcr.io/epicgames/unreal-engine:dev-slim-5.1.0  as builder
+FROM ghcr.io/epicgames/unreal-engine:dev-slim-5.1.0 as builder
```

Comment:
Maybe silly question - can we use `ghcr.io/epicgames/unreal-engine:dev-slim-5.1.0` as a build step, and do something like:

```
/home/ue4/UnrealEngine/Engine/Build/BatchFiles/RunUAT.sh BuildCookRun \
	-Server -ServerConfig=$${SERVER_CONFIG} \
	-Project=/workspace/game/Droidshooter.uproject \
	-UTF8Output -NoDebugInfo -AllMaps -NoP4 -Build -Cook -Stage -Pak -Package -Archive \
	-ArchiveDirectory=/workspace/Packaged \
	-Platform=Linux
```

As either a script of long set of build args to pass to the container?

Since then it's mounted in `/workspace` and we don't need an intermediary container image?

Then the rest of the steps for building an archive (tar.gz?) and pushing to storage just become more cloud build steps.